IFC4x3/Properties/a/AverageDepthOfBoreholes_3NM6PipTTFyvqSdj8magOF/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="AverageDepthOfBoreholes_3NM6PipTTFyvqSdj8magOF" Name="AverageDepthOfBoreholes" UniqueId="d758666c-cdd7-4ff3-9d1c-9ed23092a60f" PrimaryDataType="IfcPositiveLengthMeasure" />
```

Comment:
Propose renaming to "AverageBoreholDepth".

Reply:
Done in [98fe0b2](https://github.com/bSI-InfraRoom/IFC-Specification/pull/678/commits/98fe0b26488006ded4b93c525a4779f670f1433a)